lib/widgets/action_sheet.dart

```diff
@@ -167,10 +186,45 @@ class QuoteAndReplyButton extends MessageActionSheetMenuItemButton {
     composeBoxController.contentController
       .registerQuoteAndReplyEnd(PerAccountStoreWidget.of(messageListContext), tag,
         message: message,
-        rawContent: fetchedMessage?.content,
+        rawContent: rawContent,
       );
     if (!composeBoxController.contentFocusNode.hasFocus) {
       composeBoxController.contentFocusNode.requestFocus();
     }
   };
 }
+
+class CopyButton extends MessageActionSheetMenuItemButton {
+  CopyButton({
+    super.key,
+    required super.message,
+    required super.messageListContext,
+  });
+
+  @override get icon => Icons.copy;
+
+  @override get label => 'Copy to clipboard';
```

Comment:
This should be a bit clearer about what it's going to do, I think — there are several things one might imagine being copied.

Perhaps "Copy message text"?